sbnobj/Common/CRT/CRTTrack.hh

```diff
@@ -17,21 +17,21 @@
 #include <map>
 
 namespace sbn::crt {
-  
+
   struct CRTTrack{
 
     std::vector<uint8_t> feb_id;
     std::map< uint8_t, std::vector<std::pair<int,float> > > pesmap;
     float peshit;
```

Comment:
I suggest to add comments with descriptions to all variables. Or even better, rename variables so that they don't use obscure abbreviation, and instead are can be understood more intuitively.

Reply:
I agree, I added comments with description, and removed the two CRTTrack variables `feb_id` and `pesmap`which are used neither in `sbndcode` nor in `icaruscode`.